revpi_core.c

```diff
@@ -386,26 +451,41 @@ void revpi_core_fini(void)
 	kthread_stop(piCore_g.pUartThread);
 	piIoComm_finish();
 
-	/* reset GPIO direction */
-	piIoComm_writeSniff1A(enGpioValue_Low, enGpioMode_Input);
-	gpiod_put(piCore_g.gpio_sniff1a);
-	piIoComm_writeSniff2A(enGpioValue_Low, enGpioMode_Input);
-	gpiod_put(piCore_g.gpio_sniff2a);
-	if (piDev_g.machine_type == REVPI_CORE) {
-		piIoComm_writeSniff1B(enGpioValue_Low, enGpioMode_Input);
-		gpiod_put(piCore_g.gpio_sniff1b);
-		piIoComm_writeSniff2B(enGpioValue_Low, enGpioMode_Input);
-		gpiod_put(piCore_g.gpio_sniff2b);
-	} else if (piDev_g.machine_type == REVPI_CONNECT) {
-		gpiod_put(piCore_g.gpio_x2di);
-		gpiod_put(piCore_g.gpio_x2do);
-		gpiod_put(piCore_g.gpio_wdtrigger);
-	}
+	revpi_release_firmware(piCore_g.fw);
 
-	if (piDev_g.machine_type == REVPI_CORE)
-		gpiod_remove_lookup_table(&revpi_core_gpios);
-	else if (piDev_g.machine_type == REVPI_CONNECT)
-		gpiod_remove_lookup_table(&revpi_connect_gpios);
+	deinit_sniff_pins();
+	if (piDev_g.machine_type == REVPI_CONNECT)
+		deinit_connect_pins();
 
-	revpi_release_firmware(piCore_g.fw);
+	return 0;
+}
+
+static const struct of_device_id of_pibridge_match[] = {
+	{ .compatible = "kunbus,pibridge", },
+	{},
+};
+
+static struct platform_driver pibridge_driver = {
+	.probe		= pibridge_probe,
+	.remove 	= pibridge_remove,
+	.driver		= {
+		.name	= "revpi-core",
```

Comment:
s/revpi-core/pibridge/

Reply:
Right, this was a leftover from the previous version with a "revpi-core" instead of a "pibridge" device. Its fixed in the latest pull request, thx.